packages/react-components/react-jsx-runtime/src/jsx-runtime.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference path="types.d.ts" />
+
+import * as React from 'react';
+import { jsx as createElement, extractChildrenFromProps } from './jsx';
+
+export function jsx<P extends {}>(
+  type: React.ElementType<P>,
+  props?: P | null,
+  key?: string,
+): React.ReactElement<P> | null {
+  // extractChildrenFromProps is required since jsx signature differs from React.createElement signature
+  return createElement(type, props, ...extractChildrenFromProps(props));
```

Comment:
Is there sense to spread there (we will spread inside `createElement` anyway)? Can't we just do following:

```suggestion
  return createElement(type, props, extractChildrenFromProps(props));
```

Reply:
The only reason would be to keep the same signature as the one of the `React.createElement` method